app/assets/javascripts/subscriptions.coffee

```diff
@@ -27,3 +74,23 @@ $(document).ready ->
           element.closest('.subscriber-group').append(error)
         else
           error.insertAfter(element)
+
+
+    # method for validating query
+    $.validator.addMethod 'isValidQuery', (value, elem, params) ->
```

Comment:
add comment about event and clicking submit

Reply:
The validation triggers the same way for the required fields on SIT.  If you click on the text box, enter nothing, then try to submit it will only validate the one you were in.  I hate that behavior, but I suspect it is hard to fix.

Reply:
comments added